JDBC/OAuthBrowserFlowSampleApp/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.vertica.poc</groupId>
+    <artifactId>vertica-jdbc-oauth-methods</artifactId>
+    <version>0.1</version>
+
+    <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.vertica.jdbc</groupId>
+            <artifactId>vertica-jdbc</artifactId>
+            <version>24.3.0-0</version>
```

Comment:
This version does not exist in maven yet.  Should you use a version that is currently in maven?

Reply:
+1, 24.2.0 was just added, we could use that

Reply:
It's a tradeoff between using a version that exists and using one that doesn't have the browser flow in it. I figured we should use the version that will work with the application.

Reply:
oh, of course. You're right this is fine